src/dummy-data/team.js

```diff
@@ -2,29 +2,55 @@ import Status from '../helpers/Status';
 import { getTeamMembersByKey } from '../raw-data/preprocess_data';
 import { coursesByTeamId } from '../raw-data/preprocess_data';
 import { raw_erteam } from '../raw-data/raw_erteam';
+import { personById } from './person';
 
-function fillTeamList(teamList) {
+
+
+function createTeamList() {
+  const teamList = []
   for (let i in raw_erteam) {
     const teamData = raw_erteam[i];
     // At the moment only return team member's emp ids from first shift
     const memberList = getTeamMembersByKey(teamData.er_team_id, 0).map(member => member.emp_id);
     // Get required courses for team
     const competenceList = coursesByTeamId[teamData.er_team_id];
 
+    let status = Status.OK;

```

Comment:
Can we use `mostSevere` here? Can be imported from Status:
`import Status, { mostSevere } from '../helpers/Status';`

Reply:
something must be wrong with mostSevere. Gives this output:

console.log(mostSevere(Status.WARNING, Status.OK))
console.log(mostSevere(Status.ERROR, Status.OK))
console.log(mostSevere(Status.ERROR, Status.WARNING))
console.log(mostSevere(Status.WARNING, Status.ERROR))

Symbol(OK)
Symbol(OK)
Symbol(WARNING)
Symbol(ERROR)

Seems to return whatever is passed as second argument

Reply:
My bad. Pull the newest master changes and it should be fixed.